advanced/keyboard-shortcuts.rst

```diff
@@ -4,45 +4,136 @@ Keyboard Shortcuts
 ==================
 
 Zammad supports a wide array of keyboard shortcuts to expedite your workflow as
-an expert user.
+an expert user. But don't be afraid, you don't have to remember all of them.
 
-.. figure:: /images/advanced/keyboard-shortcuts.png
-   :alt: Keyboard shortcut cheat sheet
-   :align: center
-   :scale: 85%
-
-   The keyboard shortcut cheat sheet on Windows.
-
-.. tip:: **🖱️ UI Protip**
+.. tip:: **🤓 Have a look here to find the keyboard shortcuts**
 
    Click on your avatar at the bottom of the main menu to access the
    **keyboard shortcuts cheat sheet**.
- 
-   .. figure:: /images/extras/profile-and-settings/profile-and-settings.png
+
+   .. figure:: /images/advanced/profile-shortcuts.png
       :alt: User submenu
       :align: center
       :scale: 85%
- 
+
    Alternately, bring it up with one of the shortcuts below
    (shortcut-ception!)
```

Comment:
What does `shortcut-ception` mean? I'm not familiar with this term.

Reply:
Me neither. I search for it and it probably refers to a recursion (open shortcuts with a shortcut). I have changed it now. Probably not that much used then.